onnxruntime/python/tools/quantization/qdq_quantizer.py

```diff
@@ -234,6 +258,319 @@
             return True
         return False
 
+    def create_dynamic_subgraph(self, input_name):
+        """
+        Create nodes for dynamic quantization of input and add them to nodes_list.
+            parameter input_name: Name of the input.
+            parameter nodes_list: new nodes are appended to this list.
+            parameter qType: type to quantize to.
+            return: scale_name, zero_point_name, scale_shape, zero_point_shape.
```

Comment:
Please update the comment accordingly. nodes_list and qType don't exist for this function, and return type needs to add the 'function' node